asteroid/dsp/overlap_add.py

```diff
@@ -116,7 +128,7 @@ def ola_forward(self, x):
             padding=(self.window_size, 0),
             stride=(self.hop_size, 1),
         )
-        return out.squeeze(-1).reshape(batch, self.n_src, -1)
+        return out.squeeze(-1).reshape(batch, n_src, -1)
 
     def forward(self, x):
```

Comment:
A bit unrelated but we should be able to pass kwargs from here to the network.
With a warning that we don't segment the kwargs so they are shared between chunks.